tests/stability/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+
+def pytest_collection_modifyitems(config, items):
+    # Ensure stability tests use spot instances by default
```

Comment:
Were you able to check this works as expected? I wonder how can we check that stability is always using spot instances.

Reply:
Right now it's not easy to get that information (I learned this from @ntabris) so I just added some print statements locally to confirm that `backend_options` are getting piped correctly